packages/nuxt-locokit/src/styles/primevue/_dialog.scss

```diff
@@ -1,16 +1,17 @@
+
 .p-dialog {
   border-radius: var(--border-radius);
-  box-shadow: 0px 11px 15px -7px rgba(0, 0, 0, 0.2), 0px 24px 38px 3px rgba(0, 0, 0, 0.14), 0px 9px 46px 8px rgba(0, 0, 0, 0.12);
-  border: 0 none;
+  box-shadow: none;
+  border: 1px solid #3f4b5b;
```

Comment:
Is this color defined in vars ?
I think we need to avoid hard coded vars.

Reply:
Impossible to tell without testing the component. (cf #171)